.github/workflows/push_master.yml

```diff
@@ -47,14 +51,18 @@ jobs:
         with:
           arch: aarch64
           distro: ubuntu_latest
+          dockerRunArgs:
```

Comment:
We also need t ocreate the folder in the `setup` step: 
```
 setup: |
            mkdir -p "${PWD}/artifacts"
```
See https://github.com/uraimo/run-on-arch-action#advanced-example